ipa/ipa_gce.py

```diff
@@ -201,6 +255,7 @@ def _launch_instance(self):
         kwargs = {
             'location': self.region,
             'ex_metadata': metadata,
+            'ex_disks_gce_struct': self._get_disk_config(),
```

Comment:
drop the "_struct"

Reply:
This can't be changed as it's the Libcloud kwarg name for the GCE driver.

Reply:
http://libcloud.readthedocs.io/en/latest/compute/drivers/gce.html#libcloud.compute.drivers.gce.GCENodeDriver.create_node

Reply:
OK